contracts/ColonyNetworkMining.sol

```diff
@@ -31,9 +31,33 @@ contract ColonyNetworkMining is ColonyNetworkStorage {
     _;
   }
 
+  modifier stoppable() {
+    require(!inRepairMode, "colony-network-mining-is-in-repair-mode");
+    _;
+  }
+
+  modifier repair() {
+    require(inRepairMode, "colony-network-mining-not-in-repair-mode");
+    _;
+  }
+
+  function activateRepairMode() public auth {
+    inRepairMode = true;
+  }
+
+  function deactivateRepairMode() public auth {
```

Comment:
Regarding `auth` permissions here, we possibly need a Recovery role on a Network authority level similarly to https://github.com/JoinColony/colonyNetwork/pull/270/files#diff-bbdfafcbef1d71c176dea0989160badd